Sources/Models/Collections/AlgebraicDataTypes.swift

```diff
@@ -17,6 +17,7 @@ TODO
       whereToGoFromHere: #"""
 TODO
 """#
-    )
+    ),
+    slug: "algebraic-data-types"
```

Comment:
added this slug so that it wouldn't be `nil`. if we really do want to make it `nil` we can revert this and I can do extra `nil` checking elsewhere.